tasks.py

```diff
@@ -509,13 +513,13 @@ def ci_dnsdist_configure(c, features):
     sanitizers = ' '.join('--enable-'+x for x in os.getenv('SANITIZERS').split('+')) if os.getenv('SANITIZERS') != '' else ''
```

Comment:
I was going to suggest `os.environ.get` but it turns out `os.getenv` also takes a second `default=` argument, so either would work